xnmt/compound_expr.py

```diff
@@ -0,0 +1,13 @@
+
+class CompoundSeqExpression(object):
```

Comment:
I don't have a strong preference, either way.

Reply:
Yeah, the purpose of this data structure is mainly to convey intended semantics, so more specific sounds good.. how about even something like ```SampledExpressionSeqs``` or ```AlternativeReprExpressionSeqs```?

Reply:
It's kind of strange for SampledExpressionSeqs because to handle the part of the code that is not a list of expr_seq, I make this data structure with 1 element in it...